From 8e12840d56ec830405b5ef0ceccaece6c49dcd3d Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Fri, 14 Jul 2017 17:45:24 +0800
Subject: [PATCH 2141/5242] MLK-15989 arm: dts: imx6q: remove clk pll8_mlb

commit  016cbeaeaa9f182a50a4f4e513d49eaf7ba5a914 from
https://source.codeaurora.org/external/imx/linux-imx.git

remove clk pll8_mlb because it's not used in current driver.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl.dtsi |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 9f2d213..16d66bf 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -1157,9 +1157,8 @@
 				interrupts = <0 53 IRQ_TYPE_LEVEL_HIGH>,
 					     <0 117 IRQ_TYPE_LEVEL_HIGH>,
 					     <0 126 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX6QDL_CLK_MLB>,
-					 <&clks IMX6QDL_CLK_PLL8_MLB>;
-				clock-names = "mlb", "pll8_mlb";
+				clocks = <&clks IMX6QDL_CLK_MLB>;
+				clock-names = "mlb";
 				iram = <&ocram>;
 				status = "disabled";
 			};
-- 
1.7.9.5

